docs/manufacturer/Approved.mdx

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 2
+sidebar_label: Approved Partner Program
+---
+
+# Approved Partner Program
+
+### Introduction
+
+Betaflight is an open source firmware project, and runs on third party electronics hardware. This means that the Betaflight firmware development team does not have control over the hardware designs or specifications.
+However the performance of the firmware is related to the hardware on which it runs. Thus for the sake of the FPV community, this program aims to minimize issues that users have, by offering recommendations and guidance to manufacturers direct from the development team.
+Hardware manufacturers can now benefit from active engagement with the Betaflight development team - giving them access to their skills, experience and advice - to produce better hardware and more successful designs.
+
+### Betaflight 4.3 - Recommended Direction
+
+For version 4.3 a team of Betaflight developers started a Hardware Specification program discussing a recommended direction for current and future hardware design and development.
+The document Manufacturer Design Guidelines is available here, and represents the first time that recommendations are officially made directly from the Betaflight developers to manufacturers. We encourage manufacturers to read this document and offer feedback.
+
+### Betaflight 4.4 - A New Relationship
+
+Since releasing the Manufacturer Guidelines we have opened new communication channels with selected manufacturers within the application/website Discord. The goal was to discuss issues and engage with the companies, as well as test their hardware in order to verify the quality and performance.
+As part of the incentive to encourage engagement, we are offering a Approved Partnership official certification to manufacturers.
```

Comment:
```suggestion
As part of the incentive to encourage engagement, we are offering an Approved Partnership official certification to manufacturers.
```

Reply:
Not native English here so I might be incorrect :)

Reply:
it is correct, I switched in the last moment from Betaflight Partnership to Approved Partnership....forgot to change that a to an - well spotted...